js/greeter.js

```diff
@@ -228,6 +228,8 @@ class AntergosTheme {
 
 			if ( lightdm[ cmd ] ) {
 				$( template ).appendTo( $( this.$actions_container ) ).click( action, ( event ) => {
+					// prevent accidential hibernates after resume 
```

Comment:
Yeah that occurred to me last night while I was working on some other stuff in the theme. I decided to add a confirmation dialogue for the action buttons to address this issue.